### ISSUE TITLE
why is link to appendix broken

### ISSUE THREAD

**Issue by jbloom** [dms-vep/RABV_Pasteur_G_DMS#56]
@WillHannon-MCB, the link to the `appendix` at the bottom of this page is broken: https://dms-vep.org/RABV_Pasteur_G_DMS/

But as far as I can tell, I specified the link correctly in [this line](https://github.com/dms-vep/RABV_Pasteur_G_DMS/blob/main/homepage/index.md?plain=1#L31).

Can you let me know what is wrong in the Markdown link above?

**Comment by WillHannon-MCB** [comment]
It's fundamentally correct but needs the 'base' URL. When we specify the links in the navigation bar we also specify the 'base':

```js
// https://vitepress.dev/reference/site-config
export default defineConfig({
  lang: "en-US",
  title: "Deep mutational scanning of rabies G (Pasteur strain)",
  description:
    "Pseudovirus deep mutational scanning to measure how rabies G mutations affect cell entry and escape from a panel of monoclonal antibodies",
  base: "/RABV_Pasteur_G_DMS/", // <---- this line right here
  appearance: false,
  themeConfig: {
    // https://vitepress.dev/reference/default-theme-config
    nav: [
      { text: "Home", link: "/" },
      { text: "Appendix", link: "/appendix", target: "_self" },
    ],
    socialLinks: [{ icon: "github", link: "https://github.com/dms-vep/RABV_Pasteur_G_DMS" }],
    footer: {
      message: 'Study by Arjun Aditham, Caelan Radford, Caleb, Carr, & <a href="https://jbloomlab.org">Jesse Bloom</a>',
    },
  },
});
```

So we may need to add that to the link in the markdown:

```md
[Appendix](RABV_Pasteur_G_DMS/appendix){target="_self"}
```

This might be automatically appended to the links for pages parsed from `.md` files. I'm not sure. If so, it's because those files are handled by Vue's internal router (the tool that switches between pages) while the `.html` files included in `public/` aren't treated like "routes", they're treated like links, just internal ones.

**Comment by jbloom** [comment]
Fixed in #57, turns out needed to change `/appendix` >- `/appendix.html`, not add the prefix.
